multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -157,6 +198,13 @@ write_files:
           "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services"
       done
 
+  - path: /opt/bin/host-rkt
```

Comment:
I think it might help to have a note that this is a workaround to that issue directly in the template would be helpful. Maybe even briefly describe how it works. Looking at that file, that script is not intuitive (couple months from now we may not even remember ourselves).